cmd/manager/main.go

```diff
@@ -31,5 +33,7 @@ func main() {
 	if err != nil {
 		log.Fatal("Error building kubeconfig", err)
 	}
-	sharedmain.MainWithConfig(signals.NewContext(), "serving_operator", cfg, knativeserving.NewController)
+	ctx := signals.NewContext()
+	ctx = common.WithPlatforms(ctx, minikube.Platform)
```

Comment:
Minikube should not be referenced in the main manager. Create a minikube specific entry.